src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -292,6 +292,12 @@ void IiqDecoder::CorrectPhaseOneC(ByteStream meta_data, uint32_t split_row,
       correctSensorDefects(meta_data.getSubStream(offset, len));
       SensorDefectsSeen = true;
       break;
+    case 0x40b: // Chroma calibration
+      PhaseOneFlatField(meta_data.getSubStream(offset, len), 0, 4);
```

Comment:
1. `/*is_float=*/false` and so on
2. it is never float, let's drop that param (or should be a `bool`)
3. sink `nc` into `PhaseOneFlatField`, just pass a `bool` or better yet an `enum`

Reply:
is_float is true in dcraw's correction tag 0x401 but I found no corresponding metadata in the supported raw files, so it's gone. Can be resurrected within PhaseOneFlatField() if needed. I also followed your other suggestions.